### ISSUE TITLE
MLH_esign does not run

### ISSUE THREAD

**Issue by ekressmiller** [mplp/docassemble-mlhframework#122]
The framework has this code:
`---
code: |
  MLH_esign_supported = True

  MLH_esign = False                 
---`

If we call MLH_standard_outro_pages, I think the code block above runs (because it needs the value of MLH_esign_supported, but it has to run the whole block) and sets MLH_esign to False, so then it does not ask if you want to esign. 
  
  @normon66 I think we can just delete the "MLH_esign = False" line from the code block above. Do you see a reason not to?

**Comment by normon66** [comment]
That's probably fine, but perhaps move that line to its own block, just to denote a default for it?

```
---
code: |
  MLH_esign_supported = True         # Are electronic signatures supported?
---
code: |
  MLH_esign = False                  # Add electronic signature?
```

**Comment by ekressmiller** [comment]
To my mind, I can't think of a scenario in which MLH_esign_supported would be true but we would _not_ want to ask the MLH_esign question, so I'm not sure it's adding much to have a separate default for that, but if you think it could come up, that solution works for me!

**Comment by normon66** [comment]
I was thinking of a potential scenario where `MLH_esign_supported` was False band `MLH_esign` was referenced somewhere in a template or code but had not been initialized to False (ie that line had been removed from the framework).

Wouldn't that trigger the `MLH outro electronic signature` framework question to appear? In this case, we wouldn't want it to appear because 'MLH_esign_supported' was False.

**Comment by ekressmiller** [comment]
Got it, I see that. 

As I think about the way the code works, though, I think the MLH_esign = False code block will actually never run.

I just tested this with a version of the framework with the two separated code blocks separate as suggested above. I set MLH_esign_supported = False in my interview and then referenced MLH_esign. It asked me the esign question.

I think this is because it finds and runs the MLH_esign question _before_ it finds the code setting MLH_esign to False (because it starts from the bottom and works up). Which we want because if it went the other way it would never run the MLH_esign question. 

So I continue to think we should probably delete this line.
